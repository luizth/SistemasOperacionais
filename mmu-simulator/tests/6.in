14
10 64
9 47
10 26
1 123
12 11
13 104
10 17
0 76
7 78
13 59
11 18
10 26
7 31
5 55
13 14
1 80
3 50
4 116
4 126
6 73
0 53
7 8
1 42
2 103
2 121
13 14
12 12
6 73
2 69
11 56
6 6
3 74
8 34
7 7
13 14
8 72
0 61
8 34
3 68
13 122
8 34
7 7
8 109
8 46
5 32
11 13
5 110
7 125
5 107
6 79
13 1
7 78
7 31
0 28
4 108
13 122
9 120
8 34
0 99
1 38
8 63
4 126
4 82
0 53
12 88
5 32
11 39
3 66
2 117
3 66
6 73
4 82
2 97
1 62
4 126
7 78
11 52
1 92
12 88
4 94
9 48
1 38
1 83
2 49
12 118
6 75
10 4
5 67
6 75
10 33
9 112
13 119
13 1
1 123
6 73
5 16
5 67
3 101
7 78
8 46
11 36
2 15
10 4
7 85
8 98
9 47
2 69
9 106
4 84
5 67
11 56
6 73
10 35
6 22
12 88
4 9
3 74
2 97
9 29
10 64
8 91
1 62
2 15
8 34
3 66
3 50
10 17
8 91
9 112
12 118
2 51
13 59
13 119
13 20
11 39
4 108
4 9
3 50
7 8
4 116
10 4
6 6
7 31
2 103
1 92
12 57
4 108
11 56
2 103
9 112
0 71
12 19
11 13
8 63
3 95
2 15
4 108
4 108
5 107
3 68
0 53
13 43
7 30
1 58
9 47
1 83
5 10
3 102
1 58
10 4
6 105
5 23
9 29
11 39
6 3
1 2
9 112
13 93
13 119
5 107
1 62
9 77
1 38
12 12
3 74
0 76
13 20
0 115
2 97
5 10
2 15
12 19
11 96
0 76
2 15
12 19
2 117
2 69
2 69
