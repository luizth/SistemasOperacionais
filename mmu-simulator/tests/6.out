Process 10 accessed page 64 - frame 0
Process 9 accessed page 47 - frame 1
Process 10 accessed page 26 - frame 2
Process 1 accessed page 123 - frame 3
Process 12 accessed page 11 - frame 4
Process 13 accessed page 104 - frame 5
Process 10 accessed page 17 - frame 6
Process 0 accessed page 76 - frame 7
Memory is full! Frame 0 was replaced (page 64 - pid 10)
Process 7 accessed page 78 - frame 0
Memory is full! Frame 1 was replaced (page 47 - pid 9)
Process 13 accessed page 59 - frame 1
Memory is full! Frame 2 was replaced (page 26 - pid 10)
Process 11 accessed page 18 - frame 2
Memory is full! Frame 3 was replaced (page 123 - pid 1)
Process 10 accessed page 26 - frame 3
Memory is full! Frame 4 was replaced (page 11 - pid 12)
Process 7 accessed page 31 - frame 4
Memory is full! Frame 5 was replaced (page 104 - pid 13)
Process 5 accessed page 55 - frame 5
Memory is full! Frame 6 was replaced (page 17 - pid 10)
Process 13 accessed page 14 - frame 6
Memory is full! Frame 7 was replaced (page 76 - pid 0)
Process 1 accessed page 80 - frame 7
Memory is full! Frame 0 was replaced (page 78 - pid 7)
Process 3 accessed page 50 - frame 0
Memory is full! Frame 1 was replaced (page 59 - pid 13)
Process 4 accessed page 116 - frame 1
Memory is full! Frame 2 was replaced (page 18 - pid 11)
Process 4 accessed page 126 - frame 2
Memory is full! Frame 3 was replaced (page 26 - pid 10)
Process 6 accessed page 73 - frame 3
Memory is full! Frame 4 was replaced (page 31 - pid 7)
Process 0 accessed page 53 - frame 4
Memory is full! Frame 5 was replaced (page 55 - pid 5)
Process 7 accessed page 8 - frame 5
Memory is full! Frame 6 was replaced (page 14 - pid 13)
Process 1 accessed page 42 - frame 6
Memory is full! Frame 7 was replaced (page 80 - pid 1)
Process 2 accessed page 103 - frame 7
Memory is full! Frame 0 was replaced (page 50 - pid 3)
Process 2 accessed page 121 - frame 0
Memory is full! Frame 1 was replaced (page 116 - pid 4)
Process 13 accessed page 14 - frame 1
Memory is full! Frame 2 was replaced (page 126 - pid 4)
Process 12 accessed page 12 - frame 2
Page 73 in memory (frame 3)!
Process 6 accessed page 73 - frame 3
Memory is full! Frame 3 was replaced (page 73 - pid 6)
Process 2 accessed page 69 - frame 3
Memory is full! Frame 4 was replaced (page 53 - pid 0)
Process 11 accessed page 56 - frame 4
Memory is full! Frame 5 was replaced (page 8 - pid 7)
Process 6 accessed page 6 - frame 5
Memory is full! Frame 6 was replaced (page 42 - pid 1)
Process 3 accessed page 74 - frame 6
Memory is full! Frame 7 was replaced (page 103 - pid 2)
Process 8 accessed page 34 - frame 7
Memory is full! Frame 0 was replaced (page 121 - pid 2)
Process 7 accessed page 7 - frame 0
Page 14 in memory (frame 1)!
Process 13 accessed page 14 - frame 1
Memory is full! Frame 1 was replaced (page 14 - pid 13)
Process 8 accessed page 72 - frame 1
Memory is full! Frame 2 was replaced (page 12 - pid 12)
Process 0 accessed page 61 - frame 2
Page 34 in memory (frame 7)!
Process 8 accessed page 34 - frame 7
Memory is full! Frame 3 was replaced (page 69 - pid 2)
Process 3 accessed page 68 - frame 3
Memory is full! Frame 4 was replaced (page 56 - pid 11)
Process 13 accessed page 122 - frame 4
Page 34 in memory (frame 7)!
Process 8 accessed page 34 - frame 7
Page 7 in memory (frame 0)!
Process 7 accessed page 7 - frame 0
Memory is full! Frame 5 was replaced (page 6 - pid 6)
Process 8 accessed page 109 - frame 5
Memory is full! Frame 6 was replaced (page 74 - pid 3)
Process 8 accessed page 46 - frame 6
Memory is full! Frame 7 was replaced (page 34 - pid 8)
Process 5 accessed page 32 - frame 7
Memory is full! Frame 0 was replaced (page 7 - pid 7)
Process 11 accessed page 13 - frame 0
Memory is full! Frame 1 was replaced (page 72 - pid 8)
Process 5 accessed page 110 - frame 1
Memory is full! Frame 2 was replaced (page 61 - pid 0)
Process 7 accessed page 125 - frame 2
Memory is full! Frame 3 was replaced (page 68 - pid 3)
Process 5 accessed page 107 - frame 3
Memory is full! Frame 4 was replaced (page 122 - pid 13)
Process 6 accessed page 79 - frame 4
Memory is full! Frame 5 was replaced (page 109 - pid 8)
Process 13 accessed page 1 - frame 5
Memory is full! Frame 6 was replaced (page 46 - pid 8)
Process 7 accessed page 78 - frame 6
Memory is full! Frame 7 was replaced (page 32 - pid 5)
Process 7 accessed page 31 - frame 7
Memory is full! Frame 0 was replaced (page 13 - pid 11)
Process 0 accessed page 28 - frame 0
Memory is full! Frame 1 was replaced (page 110 - pid 5)
Process 4 accessed page 108 - frame 1
Memory is full! Frame 2 was replaced (page 125 - pid 7)
Process 13 accessed page 122 - frame 2
Memory is full! Frame 3 was replaced (page 107 - pid 5)
Process 9 accessed page 120 - frame 3
Memory is full! Frame 4 was replaced (page 79 - pid 6)
Process 8 accessed page 34 - frame 4
Memory is full! Frame 5 was replaced (page 1 - pid 13)
Process 0 accessed page 99 - frame 5
Memory is full! Frame 6 was replaced (page 78 - pid 7)
Process 1 accessed page 38 - frame 6
Memory is full! Frame 7 was replaced (page 31 - pid 7)
Process 8 accessed page 63 - frame 7
Memory is full! Frame 0 was replaced (page 28 - pid 0)
Process 4 accessed page 126 - frame 0
Memory is full! Frame 1 was replaced (page 108 - pid 4)
Process 4 accessed page 82 - frame 1
Memory is full! Frame 2 was replaced (page 122 - pid 13)
Process 0 accessed page 53 - frame 2
Memory is full! Frame 3 was replaced (page 120 - pid 9)
Process 12 accessed page 88 - frame 3
Memory is full! Frame 4 was replaced (page 34 - pid 8)
Process 5 accessed page 32 - frame 4
Memory is full! Frame 5 was replaced (page 99 - pid 0)
Process 11 accessed page 39 - frame 5
Memory is full! Frame 6 was replaced (page 38 - pid 1)
Process 3 accessed page 66 - frame 6
Memory is full! Frame 7 was replaced (page 63 - pid 8)
Process 2 accessed page 117 - frame 7
Page 66 in memory (frame 6)!
Process 3 accessed page 66 - frame 6
Memory is full! Frame 0 was replaced (page 126 - pid 4)
Process 6 accessed page 73 - frame 0
Page 82 in memory (frame 1)!
Process 4 accessed page 82 - frame 1
Memory is full! Frame 1 was replaced (page 82 - pid 4)
Process 2 accessed page 97 - frame 1
Memory is full! Frame 2 was replaced (page 53 - pid 0)
Process 1 accessed page 62 - frame 2
Memory is full! Frame 3 was replaced (page 88 - pid 12)
Process 4 accessed page 126 - frame 3
Memory is full! Frame 4 was replaced (page 32 - pid 5)
Process 7 accessed page 78 - frame 4
Memory is full! Frame 5 was replaced (page 39 - pid 11)
Process 11 accessed page 52 - frame 5
Memory is full! Frame 6 was replaced (page 66 - pid 3)
Process 1 accessed page 92 - frame 6
Memory is full! Frame 7 was replaced (page 117 - pid 2)
Process 12 accessed page 88 - frame 7
Memory is full! Frame 0 was replaced (page 73 - pid 6)
Process 4 accessed page 94 - frame 0
Memory is full! Frame 1 was replaced (page 97 - pid 2)
Process 9 accessed page 48 - frame 1
Memory is full! Frame 2 was replaced (page 62 - pid 1)
Process 1 accessed page 38 - frame 2
Memory is full! Frame 3 was replaced (page 126 - pid 4)
Process 1 accessed page 83 - frame 3
Memory is full! Frame 4 was replaced (page 78 - pid 7)
Process 2 accessed page 49 - frame 4
Memory is full! Frame 5 was replaced (page 52 - pid 11)
Process 12 accessed page 118 - frame 5
Memory is full! Frame 6 was replaced (page 92 - pid 1)
Process 6 accessed page 75 - frame 6
Memory is full! Frame 7 was replaced (page 88 - pid 12)
Process 10 accessed page 4 - frame 7
Memory is full! Frame 0 was replaced (page 94 - pid 4)
Process 5 accessed page 67 - frame 0
Page 75 in memory (frame 6)!
Process 6 accessed page 75 - frame 6
Memory is full! Frame 1 was replaced (page 48 - pid 9)
Process 10 accessed page 33 - frame 1
Memory is full! Frame 2 was replaced (page 38 - pid 1)
Process 9 accessed page 112 - frame 2
Memory is full! Frame 3 was replaced (page 83 - pid 1)
Process 13 accessed page 119 - frame 3
Memory is full! Frame 4 was replaced (page 49 - pid 2)
Process 13 accessed page 1 - frame 4
Memory is full! Frame 5 was replaced (page 118 - pid 12)
Process 1 accessed page 123 - frame 5
Memory is full! Frame 6 was replaced (page 75 - pid 6)
Process 6 accessed page 73 - frame 6
Memory is full! Frame 7 was replaced (page 4 - pid 10)
Process 5 accessed page 16 - frame 7
Page 67 in memory (frame 0)!
Process 5 accessed page 67 - frame 0
Memory is full! Frame 0 was replaced (page 67 - pid 5)
Process 3 accessed page 101 - frame 0
Memory is full! Frame 1 was replaced (page 33 - pid 10)
Process 7 accessed page 78 - frame 1
Memory is full! Frame 2 was replaced (page 112 - pid 9)
Process 8 accessed page 46 - frame 2
Memory is full! Frame 3 was replaced (page 119 - pid 13)
Process 11 accessed page 36 - frame 3
Memory is full! Frame 4 was replaced (page 1 - pid 13)
Process 2 accessed page 15 - frame 4
Memory is full! Frame 5 was replaced (page 123 - pid 1)
Process 10 accessed page 4 - frame 5
Memory is full! Frame 6 was replaced (page 73 - pid 6)
Process 7 accessed page 85 - frame 6
Memory is full! Frame 7 was replaced (page 16 - pid 5)
Process 8 accessed page 98 - frame 7
Memory is full! Frame 0 was replaced (page 101 - pid 3)
Process 9 accessed page 47 - frame 0
Memory is full! Frame 1 was replaced (page 78 - pid 7)
Process 2 accessed page 69 - frame 1
Memory is full! Frame 2 was replaced (page 46 - pid 8)
Process 9 accessed page 106 - frame 2
Memory is full! Frame 3 was replaced (page 36 - pid 11)
Process 4 accessed page 84 - frame 3
Memory is full! Frame 4 was replaced (page 15 - pid 2)
Process 5 accessed page 67 - frame 4
Memory is full! Frame 5 was replaced (page 4 - pid 10)
Process 11 accessed page 56 - frame 5
Memory is full! Frame 6 was replaced (page 85 - pid 7)
Process 6 accessed page 73 - frame 6
Memory is full! Frame 7 was replaced (page 98 - pid 8)
Process 10 accessed page 35 - frame 7
Memory is full! Frame 0 was replaced (page 47 - pid 9)
Process 6 accessed page 22 - frame 0
Memory is full! Frame 1 was replaced (page 69 - pid 2)
Process 12 accessed page 88 - frame 1
Memory is full! Frame 2 was replaced (page 106 - pid 9)
Process 4 accessed page 9 - frame 2
Memory is full! Frame 3 was replaced (page 84 - pid 4)
Process 3 accessed page 74 - frame 3
Memory is full! Frame 4 was replaced (page 67 - pid 5)
Process 2 accessed page 97 - frame 4
Memory is full! Frame 5 was replaced (page 56 - pid 11)
Process 9 accessed page 29 - frame 5
Memory is full! Frame 6 was replaced (page 73 - pid 6)
Process 10 accessed page 64 - frame 6
Memory is full! Frame 7 was replaced (page 35 - pid 10)
Process 8 accessed page 91 - frame 7
Memory is full! Frame 0 was replaced (page 22 - pid 6)
Process 1 accessed page 62 - frame 0
Memory is full! Frame 1 was replaced (page 88 - pid 12)
Process 2 accessed page 15 - frame 1
Memory is full! Frame 2 was replaced (page 9 - pid 4)
Process 8 accessed page 34 - frame 2
Memory is full! Frame 3 was replaced (page 74 - pid 3)
Process 3 accessed page 66 - frame 3
Memory is full! Frame 4 was replaced (page 97 - pid 2)
Process 3 accessed page 50 - frame 4
Memory is full! Frame 5 was replaced (page 29 - pid 9)
Process 10 accessed page 17 - frame 5
Page 91 in memory (frame 7)!
Process 8 accessed page 91 - frame 7
Memory is full! Frame 6 was replaced (page 64 - pid 10)
Process 9 accessed page 112 - frame 6
Memory is full! Frame 7 was replaced (page 91 - pid 8)
Process 12 accessed page 118 - frame 7
Memory is full! Frame 0 was replaced (page 62 - pid 1)
Process 2 accessed page 51 - frame 0
Memory is full! Frame 1 was replaced (page 15 - pid 2)
Process 13 accessed page 59 - frame 1
Memory is full! Frame 2 was replaced (page 34 - pid 8)
Process 13 accessed page 119 - frame 2
Memory is full! Frame 3 was replaced (page 66 - pid 3)
Process 13 accessed page 20 - frame 3
Memory is full! Frame 4 was replaced (page 50 - pid 3)
Process 11 accessed page 39 - frame 4
Memory is full! Frame 5 was replaced (page 17 - pid 10)
Process 4 accessed page 108 - frame 5
Memory is full! Frame 6 was replaced (page 112 - pid 9)
Process 4 accessed page 9 - frame 6
Memory is full! Frame 7 was replaced (page 118 - pid 12)
Process 3 accessed page 50 - frame 7
Memory is full! Frame 0 was replaced (page 51 - pid 2)
Process 7 accessed page 8 - frame 0
Memory is full! Frame 1 was replaced (page 59 - pid 13)
Process 4 accessed page 116 - frame 1
Memory is full! Frame 2 was replaced (page 119 - pid 13)
Process 10 accessed page 4 - frame 2
Memory is full! Frame 3 was replaced (page 20 - pid 13)
Process 6 accessed page 6 - frame 3
Memory is full! Frame 4 was replaced (page 39 - pid 11)
Process 7 accessed page 31 - frame 4
Memory is full! Frame 5 was replaced (page 108 - pid 4)
Process 2 accessed page 103 - frame 5
Memory is full! Frame 6 was replaced (page 9 - pid 4)
Process 1 accessed page 92 - frame 6
Memory is full! Frame 7 was replaced (page 50 - pid 3)
Process 12 accessed page 57 - frame 7
Memory is full! Frame 0 was replaced (page 8 - pid 7)
Process 4 accessed page 108 - frame 0
Memory is full! Frame 1 was replaced (page 116 - pid 4)
Process 11 accessed page 56 - frame 1
Page 103 in memory (frame 5)!
Process 2 accessed page 103 - frame 5
Memory is full! Frame 2 was replaced (page 4 - pid 10)
Process 9 accessed page 112 - frame 2
Memory is full! Frame 3 was replaced (page 6 - pid 6)
Process 0 accessed page 71 - frame 3
Memory is full! Frame 4 was replaced (page 31 - pid 7)
Process 12 accessed page 19 - frame 4
Memory is full! Frame 5 was replaced (page 103 - pid 2)
Process 11 accessed page 13 - frame 5
Memory is full! Frame 6 was replaced (page 92 - pid 1)
Process 8 accessed page 63 - frame 6
Memory is full! Frame 7 was replaced (page 57 - pid 12)
Process 3 accessed page 95 - frame 7
Memory is full! Frame 0 was replaced (page 108 - pid 4)
Process 2 accessed page 15 - frame 0
Memory is full! Frame 1 was replaced (page 56 - pid 11)
Process 4 accessed page 108 - frame 1
Page 108 in memory (frame 1)!
Process 4 accessed page 108 - frame 1
Memory is full! Frame 2 was replaced (page 112 - pid 9)
Process 5 accessed page 107 - frame 2
Memory is full! Frame 3 was replaced (page 71 - pid 0)
Process 3 accessed page 68 - frame 3
Memory is full! Frame 4 was replaced (page 19 - pid 12)
Process 0 accessed page 53 - frame 4
Memory is full! Frame 5 was replaced (page 13 - pid 11)
Process 13 accessed page 43 - frame 5
Memory is full! Frame 6 was replaced (page 63 - pid 8)
Process 7 accessed page 30 - frame 6
Memory is full! Frame 7 was replaced (page 95 - pid 3)
Process 1 accessed page 58 - frame 7
Memory is full! Frame 0 was replaced (page 15 - pid 2)
Process 9 accessed page 47 - frame 0
Memory is full! Frame 1 was replaced (page 108 - pid 4)
Process 1 accessed page 83 - frame 1
Memory is full! Frame 2 was replaced (page 107 - pid 5)
Process 5 accessed page 10 - frame 2
Memory is full! Frame 3 was replaced (page 68 - pid 3)
Process 3 accessed page 102 - frame 3
Page 58 in memory (frame 7)!
Process 1 accessed page 58 - frame 7
Memory is full! Frame 4 was replaced (page 53 - pid 0)
Process 10 accessed page 4 - frame 4
Memory is full! Frame 5 was replaced (page 43 - pid 13)
Process 6 accessed page 105 - frame 5
Memory is full! Frame 6 was replaced (page 30 - pid 7)
Process 5 accessed page 23 - frame 6
Memory is full! Frame 7 was replaced (page 58 - pid 1)
Process 9 accessed page 29 - frame 7
Memory is full! Frame 0 was replaced (page 47 - pid 9)
Process 11 accessed page 39 - frame 0
Memory is full! Frame 1 was replaced (page 83 - pid 1)
Process 6 accessed page 3 - frame 1
Memory is full! Frame 2 was replaced (page 10 - pid 5)
Process 1 accessed page 2 - frame 2
Memory is full! Frame 3 was replaced (page 102 - pid 3)
Process 9 accessed page 112 - frame 3
Memory is full! Frame 4 was replaced (page 4 - pid 10)
Process 13 accessed page 93 - frame 4
Memory is full! Frame 5 was replaced (page 105 - pid 6)
Process 13 accessed page 119 - frame 5
Memory is full! Frame 6 was replaced (page 23 - pid 5)
Process 5 accessed page 107 - frame 6
Memory is full! Frame 7 was replaced (page 29 - pid 9)
Process 1 accessed page 62 - frame 7
Memory is full! Frame 0 was replaced (page 39 - pid 11)
Process 9 accessed page 77 - frame 0
Memory is full! Frame 1 was replaced (page 3 - pid 6)
Process 1 accessed page 38 - frame 1
Memory is full! Frame 2 was replaced (page 2 - pid 1)
Process 12 accessed page 12 - frame 2
Memory is full! Frame 3 was replaced (page 112 - pid 9)
Process 3 accessed page 74 - frame 3
Memory is full! Frame 4 was replaced (page 93 - pid 13)
Process 0 accessed page 76 - frame 4
Memory is full! Frame 5 was replaced (page 119 - pid 13)
Process 13 accessed page 20 - frame 5
Memory is full! Frame 6 was replaced (page 107 - pid 5)
Process 0 accessed page 115 - frame 6
Memory is full! Frame 7 was replaced (page 62 - pid 1)
Process 2 accessed page 97 - frame 7
Memory is full! Frame 0 was replaced (page 77 - pid 9)
Process 5 accessed page 10 - frame 0
Memory is full! Frame 1 was replaced (page 38 - pid 1)
Process 2 accessed page 15 - frame 1
Memory is full! Frame 2 was replaced (page 12 - pid 12)
Process 12 accessed page 19 - frame 2
Memory is full! Frame 3 was replaced (page 74 - pid 3)
Process 11 accessed page 96 - frame 3
Page 76 in memory (frame 4)!
Process 0 accessed page 76 - frame 4
Page 15 in memory (frame 1)!
Process 2 accessed page 15 - frame 1
Page 19 in memory (frame 2)!
Process 12 accessed page 19 - frame 2
Memory is full! Frame 4 was replaced (page 76 - pid 0)
Process 2 accessed page 117 - frame 4
Memory is full! Frame 5 was replaced (page 20 - pid 13)
Process 2 accessed page 69 - frame 5
Page 69 in memory (frame 5)!
Process 2 accessed page 69 - frame 5
